application/controllers/app.rb

```diff
@@ -371,6 +373,34 @@ class App < Roda
       routing.on 'analytics' do
         @current_account = SecureSession.new(session).get(:current_account)
 
+        # routing.redirect '/' unless @current_account
+
+        # GET /analytics/download_logs
+        routing.is 'download_logs' do
+          routing.get do
+            puts 'Accessing /analytics/download_logs...'
+
+            file_path = routing.params['file']
+            puts "Received file request for: #{file_path}"
+
+            result = Service::GetLogFileDownload.new.call(file_path)
+
+            if result.failure?
+              response.status = 500
+              response.write({ error: "Log processing failed." }.to_json)
+            else
+              full_path = File.join('application/public', file_path)
+              response['Content-Type'] = 'application/octet-stream'
+              response['Content-Disposition'] = "attachment; filename=\"#{File.basename(file_path)}\""
+              response.write(File.read(full_path))
+
+              folder_path = File.join(__dir__, "../public")
+              FileUtils.remove_dir(folder_path)
+              puts "Folder #{folder_path} has been deleted."
```

Comment:
I think this `puts` can be removed? I don't see anything in this `else` block that could fail.